### ISSUE TITLE
add a manpage

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#44]
```
skipfish is distributed without manpage, which is quite popular format of
documentation. One of Debian developers created such manpage and it is
available in Debian's BTS:

http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=575596

Would be great if you could include it in your tarball.

regards
fEnIo
```

Original issue reported on code.google.com by `fen...@gmail.com` on 28 Mar 2010 at 10:08

**Comment by GoogleCodeExporter** [comment]
```
Yup, thanks. I wanted to tweak it a bit, but will include.
```

Original comment by `lcam...@gmail.com` on 28 Mar 2010 at 5:37
- Changed title: **add a manpage**
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.27. Thanks.
```

Original comment by `lcam...@gmail.com` on 30 Mar 2010 at 5:53
- Changed state: **Fixed**
